packages/adapter-axios/src/index.ts

```diff
@@ -30,7 +18,7 @@ export class AxiosAdapter {
 		})
 	}
 
-	async request<T = any>(method: string, path: string, body?: any, query?: any, options?: AxiosRequestConfig): Promise<AdapterResponse<T>> {
+	async request<T = any>(method: string, path: string, body: unknown, query: Record<string, string>, options: AxiosRequestConfig): Promise<AdapterResponse<T>> {
```

Comment:
Query type `Record<string, string>` is not correct, because as value can be passed also boolean, number and array.

Reply:
Good point
I will also try to find a similar solution for the Fetch adapter